### PR TITLE
Add classnames to dashboard widget

### PR DESCRIPTION
**Pull request by Xyfi**
## Summary

This PR can be summarized in the following changelog entry:

* Adds class names to `WordpressFeed` and `SeoAssessment` components, and also to their children.

## Test instructions

This PR can be tested by following these steps:

* Run the development environment and navigate to the Dashboard widget tab
* Inspect the elements to see the added classnames

### CODE REVIEW

File: composites/Plugin/DashboardWidget/components/SeoAssessment.js [hunk 1]
```diff
@@ -27,22 +27,27 @@ const SeoAssessmentText = styled.p`
  */
 const SeoAssessment = ( props ) => {
 	return (
-		<SeoAssessmentContainer>
-			<SeoAssessmentText>
+		<SeoAssessmentContainer
+			className={ props.className }>
+			<SeoAssessmentText
+				className={ `${ props.className }__text`}>
```
Comment (boblinthorst) [inline composites/Plugin/DashboardWidget/components/SeoAssessment.js]: space at last curly brace

File: composites/Plugin/DashboardWidget/components/WordpressFeed.js [hunk 2]
```diff
@@ -73,8 +73,10 @@ const WordpressFeedFooter = styled.div`
 
 const WordpressFeedListItem = ( props ) => {
 	return (
-		<WordpressFeedListItemContainer>
+		<WordpressFeedListItemContainer
```
Comment (boblinthorst) [inline composites/Plugin/DashboardWidget/components/WordpressFeed.js]: is technically WordPress, not sure how impactful it is to change this, but if it's easy to change, please do :)

File: composites/Plugin/DashboardWidget/components/WordpressFeed.js [hunk 3]
```diff
@@ -83,12 +85,15 @@ const WordpressFeedListItem = ( props ) => {
 					( Opens in a new browser tab )
 				</A11yNotice>
 			</WordpressFeedLink>
-			<p>{ props.description }</p>
+			<p className={ `${props.className}-description` }>
```
Comment (boblinthorst) [inline composites/Plugin/DashboardWidget/components/WordpressFeed.js]: spaces in inner curly braces.

File: composites/Plugin/DashboardWidget/components/WordpressFeed.js [hunk 4]
```diff
@@ -107,24 +112,30 @@ WordpressFeedListItem.propTypes = {
  */
 const WordpressFeed = ( props ) => {
 	return (
-		<WordpressFeedContainer>
-			<WordpressFeedHeader>
+		<WordpressFeedContainer
+			className={ props.className }>
+			<WordpressFeedHeader
+				className={ `${props.className}__header` }>
```
Comment (boblinthorst) [inline composites/Plugin/DashboardWidget/components/WordpressFeed.js]: space at inner curly brace.

Comment (boblinthorst) [inline composites/Plugin/DashboardWidget/components/WordpressFeed.js]: space at inner curly brace.

File: composites/Plugin/DashboardWidget/components/WordpressFeed.js [hunk 5]
```diff
@@ -107,24 +112,30 @@ WordpressFeedListItem.propTypes = {
...
 						link={ item.link }
 						description={ item.description } />
 				) ) }
 			</WordpressFeedList>
 			{ props.footerHtml &&
-				<WordpressFeedFooter>
-					<Divider />
+				<WordpressFeedFooter
+					className={ `${props.className}__footer` }>
+					<Divider className={ `${props.className}__footer-divider` }/>
```
Comment (boblinthorst) [inline composites/Plugin/DashboardWidget/components/WordpressFeed.js]: curly braces

File: composites/Plugin/DashboardWidget/components/WordpressFeed.js [hunk 6]
```diff
@@ -107,24 +112,30 @@ WordpressFeedListItem.propTypes = {
...
 					<WordpressFeedLink
+						className={ `${props.className}__footer-link` }
```
Comment (boblinthorst) [inline composites/Plugin/DashboardWidget/components/WordpressFeed.js]: curly braces

File: composites/Plugin/DashboardWidget/components/WordpressFeed.js [hunk 7]
```diff
@@ -73,8 +73,10 @@ const WordpressFeedFooter = styled.div`
 
 const WordpressFeedListItem = ( props ) => {
 	return (
-		<WordpressFeedListItemContainer>
+		<WordpressFeedListItemContainer
+			className={ props.className }>
 			<WordpressFeedLink
+				className={ `${props.className}-link` }
```
Comment (boblinthorst) [inline composites/Plugin/DashboardWidget/components/WordpressFeed.js]: this one too :)

File: composites/Plugin/DashboardWidget/components/WordpressFeed.js (hunk 2 above)
Comment (boblinthorst) [reply composites/Plugin/DashboardWidget/components/WordpressFeed.js]: I see this was not introduced in this pull, so I won't block it based on it. Though we might want to change it in the future.

File: composites/Plugin/DashboardWidget/components/SeoAssessment.js [hunk 8]
```diff
@@ -27,22 +27,27 @@ const SeoAssessmentText = styled.p`
...
-		<SeoAssessmentContainer>
-			<SeoAssessmentText>
+		<SeoAssessmentContainer
+			className={ props.className }>
+			<SeoAssessmentText
+				className={ `${ props.className }__text` }>
 				{ props.seoAssessmentText }
 			</SeoAssessmentText>
 			<StackedProgressBar
+				className={ "progress" }
```
Comment (herregroen) [inline composites/Plugin/DashboardWidget/components/SeoAssessment.js]: Why isn't this passed as `className="progress"`?

File: composites/Plugin/DashboardWidget/components/SeoAssessment.js [hunk 9]
```diff
@@ -27,22 +27,27 @@ const SeoAssessmentText = styled.p`
...
 				items={ props.seoAssessmentItems }
 				barHeight={ props.barHeight }
-				/>
+			/>
 			<SeoScoreAssessments
+				className={ "assessments" }
```
Comment (herregroen) [inline composites/Plugin/DashboardWidget/components/SeoAssessment.js]: See above.

File: composites/Plugin/Shared/tests/__snapshots__/StackedProgresBarTest.js.snap [hunk 10]
```diff
@@ -5,16 +5,16 @@ exports[`the BaseButton matches the snapshot 1`] = `
   className="sc-bdVaJa bJbZBf"
 >
   <span
-    className="sc-bwzfXH gpLKCA"
+    className="undefined-part sc-bwzfXH gpLKCA"
```
Comment (herregroen) [inline composites/Plugin/Shared/tests/__snapshots__/StackedProgresBarTest.js.snap]: This doesn't seem handy to have in a snapshot. A `className` should be added to the test.

**Comment by herregroen** [comment]
CR done.

**Comment by IreneStr** [comment]
CR done 👍 
I've corrected the test description of the stacked progress bar.

**Comment by IreneStr** [comment]
Acceptance done
Please have a look at the dividers

**Comment by IreneStr** [comment]
CR and acceptance done 👍

**Pull request merged**